Common/include/CConfig.hpp

```diff
@@ -80,6 +80,7 @@ class CConfig {
   su2double EA_ScaleFactor;       /*!< \brief Equivalent Area scaling factor */
   su2double AdjointLimit;         /*!< \brief Adjoint variable limit */
   string* ConvField;              /*!< \brief Field used for convergence check.*/
+  string FLUID_NAME;              /*!< \brief name of the applied fluid. */
```

Comment:
Please don't introduce new styles to the code... only enums are all capitals in SU2
```suggestion
  string FluidName;               /*!< \brief name of the applied fluid. */
```

Reply:
I changed the name as you advised, thank you.